atqa-framework/tests/integration/scenario.Test.ts

```diff
@@ -0,0 +1,68 @@
+import { ClockUnit } from '../../utils/clockUnit';
+import { BasePage } from '../pages/BasePage';
+import { ExtensionInitHandler } from '../../utils/ExtensionInitHandler';
+import { AccountPage } from '../pages/AccountPage';
+
+const basePage = new BasePage();
+const clockUnit = new ClockUnit();
+const extensionInitHandler = new ExtensionInitHandler();
+const accountPage = new AccountPage();
+
+const { I } = inject();
+
+Feature('Initial Test');
+
+Background(async () => {
+  return extensionInitHandler.extensionInit();
+});
+
+Scenario('Init scenario', async () => {
+  //I.wait(clockUnit.MINUTES * 30);
+  I.waitForEnabled(
+    accountPage.SELECTORS.GET_STARTED_BUTTON,
+    clockUnit.SECONDS * 30
+  );
+  I.doubleClick(accountPage.SELECTORS.GET_STARTED_BUTTON);
+  I.wait(clockUnit.SECONDS * 10);
```

Comment:
Using `wait` is bad pattern.

Reply:
This scenario for debugging.

Reply:
Okay, but don't forget to delete it afterwards.